### DISCUSSION TITLE
".NET performance" has no values for coreclr apps

### DISCUSSION THREAD

**Discussion by NKnusperer** [winsiderss/systeminformer#1141] [Questions / Answers] [answered]
Hi everyone, I've noticed that the ".NET performance" tab always reports no values for .NET Core / 5 / 6 applications using the nighty build (15436cf).
For example this is PowerShell 7.2 which is build using .NET 6.0:
![image](https://user-images.githubusercontent.com/179573/155497698-bbce34e9-8b94-4895-87eb-fd17b7a5c13e.png)

The ".NET assemblies" tab seems to be working as expected.
I couldn't find an existing issue / discussion related to this topic so I just want to ask if this is somehow expected behavior because this is not implemented yet or a bug.

Thank you :)

**Comment by dmex** [comment, marked as answer]
>  just want to ask if this is somehow expected behavior

Microsoft removed support for performance counters in .NET core and replaced them with a custom data protocol over loopback network socket: https://docs.microsoft.com/en-us/dotnet/core/diagnostics/event-counters

Process Explorer and other projects also don't support .NET core because the development required (compression libraries, parser libraries, potential security issues) for their custom protocol requires too much work. The only tool available is Microsoft's dotnet-counters: https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-counters
